src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -843,6 +843,27 @@ def playbook_artifact_replay(
                 return messages, exit_messages
         return messages, exit_messages
 
+    @staticmethod
+    @_post_processor
+    def pull_arguments(
+        entry: SettingsEntry,
+        config: ApplicationConfiguration,
+    ) -> PostProcessorReturn:
+        # pylint: disable=unused-argument
+        """Post process ``pull_arguments``
+
+        :param entry: The current settings entry
+        :param config: The full application configuration
+        :return: An instance of the standard post process return object
+        """
+        messages: List[LogMessage] = []
+        exit_messages: List[ExitMessage] = []
+        if entry.value.source == C.ENVIRONMENT_VARIABLE:
+            entry.value.current = to_list(entry.value.current)
+        if entry.value.current is not C.NOT_SET:
```

Comment:
If the value is not set can it be initialized to an empty list to avoid type checking later on in the code?

Reply:
We don't currently have a good way to do this, the default would have to be set to `[]` which doesn't translate well to command line parameters, if we didn't use the default, it would be odd in the `:settings` subcommand because the current would be different from the default, would show as "changed" but we don't have a Constant to describe "why" it was changed.